sympy/sets/sets.py

```diff
@@ -534,6 +535,72 @@ def measure(self):
         """
         return self._measure
 
+    @property
+    def kind(self):
+        """
+        The kind of a Set
+
+        Explanation
+        ===========
+
+        Any :py:class`~.Set` will have kind :py:class`~.SetKind` which is
```

Comment:
If you actually build the docs as described in the docs guide you'll see that these references have not turned into hyperlinks. That's because of the missing colons so it should be e.g.
```
:py:class:`~.Set`
```
If you fix that then you will see this error when building the docs:
```python
sympy/sets/sets.py:docstring of sympy.sets.sets.Set.kind:9: WARNING: py:class reference target not found: SetKind
```
That's because SetKind isn't actually included in the docs yet. A reference to it should be added in `doc/src/modules/sets.rst`.

You should make sure also that `TupleKind` shows up in the docs somewhere. Also check through all of the docstrings you have written: build the docs and see that everything is correctly rendered and that the hyperlinks all work including in the see also sections.